recipes/intel_install.rb

```diff
@@ -18,42 +18,47 @@
 case node['cfncluster']['cfn_node_type']
 when 'MasterServer'
 
+  # Install yum4 needed to handle Intel Packages
+  package 'nextgen-yum4' do
+    retries 3
+    retry_delay 5
+  end
+
   # Downloads the intel-hpc-platform rpms
   bash "install intel hpc platform" do
     cwd node['cfncluster']['sources_dir']
     code <<-INTEL
       set -e
       yum-config-manager --add-repo http://yum.repos.intel.com/hpc-platform/el7/setup/intel-hpc-platform.repo
-      yum-config-manager --save --setopt=intel-hpc-platform.skip_if_unavailable=true
+      # yum-config-manager --save --setopt=intel-hpc-platform.skip_if_unavailable=True
```

Comment:
what about removing this commented lines or at the very least let's leave a comment on why we keep them there and when to uncomment